src/components/Calculator.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react';
+
+import {
+  Container,
+  Row,
+  Col,
+  Button,
+} from 'react-bootstrap';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      result: 0,
+    };
+  }
+
+  render() {
+    const { result } = this.state;
+    return (
+      <Container>
+        <Row className="m-2">
+          <Col>
+            <div dir="rtl" className="w-100 p-2 result">
+              { result }
+            </div>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">AC</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">+/-</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">%</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4 btn-operations">/</Button>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">7</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">8</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">9</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4 btn-operations">
+              x
+            </Button>
+          </Col>
+        </Row>
+        <Row className="m-2">
+          <Col>
+            <Button className="w-100 p-4">4</Button>
+          </Col>
+          <Col>
+            <Button className="w-100 p-4">5</Button>
```

Comment:
Please ensure that the `Calculator.js` component should match the following given design (the width of your design is bigger than the provided design.).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/158899124-f6b0c345-a767-469a-96c2-6728bba5d934.png) | ![image](https://user-images.githubusercontent.com/10332499/158899191-562466a2-d163-4cc4-8352-5e264c062ac8.png)